crowdin.yml

```diff
@@ -0,0 +1,10 @@
+project_id_env: "CROWDIN_PROJECT_ID"
+api_token_env: "CROWDIN_PERSONAL_TOKEN"
+base_path_env: "CROWDIN_BASE_PATH"
+base_url_env: "CROWDIN_BASE_URL"
```

Comment:
these have to be static? is it right?

Reply:
Yes, it is. it's just mapping the environment variables in the action